apps/webpack.js

```diff
@@ -1,7 +1,12 @@
 var _ = require('lodash');
 var webpack = require('webpack');
 var path = require('path');
-var config = {
+var LiveReloadPlugin = require('webpack-livereload-plugin');
+
+var AUTO_RELOAD = ['true', '1'].indexOf(process.env.AUTO_RELOAD) !== -1;
```

Comment:
Yeah, it probably should be. Originally I think I thought I was going to get away with only needing it in one of the files. Right move is probably to check for this in the gruntfile, and pass it as config to webpack's `create`.

I think we handle them inconsistently. There are some places we check for exactly 1. Here for some reason I decided to also allow for true. Maybe we are better off just checking for existence everywhere (and perhaps just providing a helper method - something like `envVariableSet('AUTO_RELOAD');`).

Thoughts?